firmware/rhizome.ino

```diff
@@ -32,6 +33,9 @@ std::deque< Ohmbrewer::Equipment* > sprouts;
  */
 int inUse[6] = {0,0,0,0,0,0};
 
+std::list<int>* thermPins = new std::list<int>(1,0);
```

Comment:
Change this to:
```c++
std::list<int> thermPins (1, 0);
```
or, alternatively, you can load up all the pins in 2 lines (like a boss):
```c++
int thmP[] = {0,4,5};
std::list<int> thermPins (thmP, thmP + sizeof(thmP) / sizeof(int));
```